pallets/gear-eth-bridge/src/tests.rs

```diff
@@ -173,8 +178,15 @@ fn bridge_send_eth_message_works() {
         assert_eq!(MessageNonce::get(), 0.into());
         assert!(Queue::get().is_empty());
 
+        // Send a message via the pallet extrinsic
+
         let destination = H160::random();
         let payload = H256::random().as_bytes().to_vec();
+        let mut gas_meter = GasSpentMeter::start();
+        let mut signer_balance = balance_of(&SIGNER);
```

Comment:
via pallet extrinsic you may only assert event of balances::transfer

Reply:
Fixed